packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx

```diff
@@ -118,6 +120,17 @@ const getDocs: () => JSX.Element = () => {
         <CustomStyleVideoGalleryExample />
       </Canvas>
 
+      <Heading>Local Video Camera Button</Heading>
+      <DetailedBetaBanner />
+      <Description>
+        The Video Gallery can take in customization to allow for the introduction of local camera controls where the
+        button will cycle through the different camera's in the users device. Typical usage is to trigger this button on
+        mobile devices.
+      </Description>
+      <Canvas>
```

Comment:
did we not want to post the source as well?

Reply:
I'm not sure what you mean? of where the gifs came from?

Reply:
sorry just figured it out. will add it!